pkg/workload/tpcc/tpcc.go

```diff
@@ -114,6 +122,10 @@ type tpcc struct {
 		values [][]int
 	}
 	localsPool *sync.Pool
+
+	// In cases of longDurationWorkloadThreshold, we skip 3.3.2.1 and 3.3.2.8
```

Comment:
nit: can be reworded to `Set to true if duration >= longDurationWorkloadThreshold. In that case, we reset ... periodically and skip .... consistency checks`.